python/ql/src/semmle/python/frameworks/Flask.qll

```diff
@@ -347,17 +347,9 @@ module Flask {
    */
   private class FlaskRequestAdditionalTaintStep extends TaintTracking::AdditionalTaintStep {
```

Comment:
Did you want to rename this class also?

Reply:
seems like I did this in a later commit :shrug:

Reply:
Seems so :-)